physics/pressure.py

```diff
@@ -0,0 +1,94 @@
+"""
+Description :
+https://en.wikipedia.org/wiki/Pressure
+
+- Solid Pressure
+P = F/A
+P is the pressure,
+F is the magnitude of the normal force,
+A is the area of the surface on contact.
+
+- Liquid Pressure
+https://en.wikipedia.org/wiki/Pressure#Liquid_pressure
+P = p*g*h
+P is liquid pressure,
+g is gravity at the surface of overlaying material,
+ρ is density of liquid,
+h is height of liquid column or depth within a substance.

```

Comment:
```suggestion
P = ρ*g*h
P is liquid pressure,
g is gravity at the surface of overlaying material,
ρ is density of liquid,
h is height of liquid column or depth within a substance.
```

Reply:
To make these changes, you need to specify the file's encoding to be utf-8. 

Because Not many would notice it until run-time. Even though this encoding may be the default for your IDE, and obvious for you -  it may not be so for others.

Kind regards,
IdoErel

Reply:
There's ρ in on line 16 already, but it's 'p' on line 13.

Reply:
Ok, you're right - keeping the docs consistent. Yet this should be addressed anyways.

Even I missed this simple special character in there. This is the kind of stuff that should be resolved ahead of time.

Many thanks,
IdoErel